main.c

```diff
@@ -95,7 +97,7 @@ int main(int argc, char **argv) {
             if (bka[0]) printbk(bks[0],  0, 12, 3);
             if (bka[1]) printbk(bks[1],  5, 12, 4);
             if (bka[2]) printbk(bks[2], 10, 12, 5);
-            printbk(bks[b], x, y, b+3);
+            if (b <3)   printbk(bks[b], x, y, b+3);
```

Comment:
heh, `<3` is cute, but maybe use `-1` for no selection, and `b >= 0` for the test (I think `3` seems to much of a legal value)